pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+**Checklist for the submission of new PEtab problems**
+- [ ] The problem ID is in the format `{LAST_NAME_OF_FIRST_AUTHOR}_{ABBREVIATED_JOURNAL_NAME}{YEAR_OF_PUBLICATION}`
+- [ ] There is a GitHub issue for this problem
+  - [ ] The problem ID is in the title
+  - [ ] A brief model description (one or two sentences)
+  - [ ] A brief data description (one or two sentences)
+  - [ ] The issue and PR are linked to each other
+  - [ ] Differences between the implementation and the original publication are described
```

Comment:
Should that be in the issue? It might be preferable to have it in the SBML annotations?